package.json

```diff
@@ -0,0 +1,64 @@
+{
+	"name": "algobuilder-msig",
+	"description": "Multisignature UI",
+	"homepage": "https://algobuilder.dev",
+	"license": "mpl-2.0",
+	"engines": {
+		"node": ">=14.11.0",
+		"yarn": ">=3.1.0"
+	},
+	"scripts": {
+		"dev": "vue-cli-service serve",
+		"build": "vue-cli-service build",
+		"lint": "yarn lint:js && yarn lint:prettier",
+		"start": "node .output/server/index.mjs",
+		"lint:js": "eslint --ext \".js,.ts,.vue\" --ignore-path .gitignore .",
+		"lint:prettier": "prettier --check .",
+		"lint:fix": "prettier --write --list-different . && yarn lint:js --fix",
+		"upgrade:dependency": "npx yarn-upgrade-all",
+		"test": "echo 'no test'"
+	},
+	"dependencies": {
+		"@algo-builder/algob": "^4.0.0",
+		"@algo-builder/web": "^4.0.0",
+		"@ant-design/icons-vue": "^6.1.0",
+		"algosdk": "^1.16.0",
+		"ant-design-vue": "^3.2.5",
+		"bootstrap": "^5.1.3",
+		"bootstrap-vue": "^2.22.0",
+		"core-js": "^3.21.1",
+		"cors": "^2.8.5",
+		"crypto-browserify": "^3.12.0",
+		"pinia": "^2.0.12",
+		"stream-browserify": "^3.0.0",
+		"vue": "^3.2.37",
+		"vue-router": "^4.0.14"
+	},
+	"lint-staged": {
+		"*.{js,ts,vue}": "eslint --cache",
+		"*.**": "prettier --ignore-unknown --write"
+	},
+	"packageManager": "yarn@3.2.0",
+	"devDependencies": {
+		"@babel/core": "^7.17.7",
+		"@babel/eslint-parser": "^7.17.0",
+		"@typescript-eslint/eslint-plugin": "^5.3.1",
+		"@typescript-eslint/parser": "^5.3.1",
+		"@vue/cli-plugin-babel": "^5.0.1",
+		"@vue/cli-plugin-eslint": "^5.0.1",
+		"@vue/cli-plugin-typescript": "^5.0.1",
+		"@vue/cli-service": "^5.0.1",
+		"@vue/compiler-sfc": "^3.2.31",
+		"@vue/eslint-config-typescript": "^10.0.0",
+		"eslint": "^8.11.0",
+		"eslint-config-prettier": "^8.5.0",
+		"eslint-plugin-vue": "^8.5.0",
+		"husky": "^7.0.4",
+		"jest": "^27.5.1",
+		"lint-staged": "^12.3.5",
+		"nodemon": "^2.0.16",
```

Comment:
`nodemon, ts-node, cors` are also needed in this project. Please check this.
Also, you are using both ant design and bootstrap library. Do you need both?

Reply:
You were supposed to remove this dependency and check other.